client/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "cSpell.words": [
+        "DATAGENE",
+        "KINETICUT",
+        "MOMENTIA",
+        "OHMNET"
+    ]
+}
```

Comment:
This file seems to have been created erroneously. There shouldn't be a separate vscode settings file just within client.

Reply:
Good point – I'll clean that up.